docs/devhub/content-syntax-guide.md

```diff
@@ -0,0 +1,238 @@
+
+This is a link to the [mermaid diagrams](content-syntax-guide.md#mermaid-diagrams) section within this page.
+
+This is a link to the [Get help in Rocket.Chat](../rocketchat/get-help-in-rocketchat.md) page in the Rocket.Chat section.
+
+This is a link to the [meetups](../../../mobile-developer-guide/meetups/) page in the Mobile Developer Guide. It is a page in another guide on DevHub. 
+
+```
+
+This is a link to the [DevHub Content Partner Guide](content-partner-guide.md) page in the DevHub category.
+
+This is a link to the [mermaid diagrams](content-syntax-guide.md#mermaid-diagrams) section within this page.
+
+This is a link to the [Get help in Rocket.Chat](../rocketchat/get-help-in-rocketchat.md) page in the Rocket.Chat section.
+
+This is a link to the [meetups](../../../mobile-developer-guide/meetups/) page in the Mobile Developer Guide. It is a page in another guide on DevHub. 
+
+## Lists
+
+There must be a line break between the sentence and the start of the list. Use indents for nested lists. 
+
+```markdown
+This is a numbered list. Using the `1.` for all items will automatically number the list for you.
+
+1. Item 1
+1. Item 2
+  1. Item 2a
+  1. Item 2b
+    1. Item 2bi
+1. Item 3
+  1. Item 3a
+```
+
+This is a numbered list. Using the `1.` for all items will automatically number the list for you.
+
+1. Item 1
+1. Item 2
+  1. Item 2a
+  1. Item 2b
+    1. Item 2bi
+1. Item 3
+  1. Item 3a
+
+Unordered lists can use either the `*` or `-` characters.
+
+```markdown
+This is an unordered list:
+
+- Item 1
+- Item 2
+- Item 3
+  - Item 3a
+  - Item 3b
+```
+
+This is an unordered list:
+
+- Item 1
+- Item 2
+- Item 3
+  - Item 3a
+  - Item 3b
+
+!!! note
+    Whitespace at the start of the top level items will prevent the list from being generated. 
+
+This list will not display properly because it has whitespace in front of each top level item:   
+
+```markdown
+  - Item 1
+  - Item 2
+    - Item 2a
+  - Item 3
+```
+ This list will not display properly because it has whitespace in front of each top level item:   
+
+  - Item 1
+  - Item 2
+    - Item 2a
+  - Item 3 
+
+## Code 
+
+Use three backticks for codeblocks. Provide the language to include syntax highlighting.
+
+````markdown
+```python
+x = 2
+if x == 1:
+    print("x is 1.")
+else:
+    print("x is not 1")
+```    
+````
+
+```python
+x = 2
+if x == 1:
+    print("x is 1.")
+else:
+    print("x is not 1")
+```
+
+```
+Use a single backtick for `inline code blocks`.
+```
+
+Use a single backtick for `inline code blocks`.
+
+
+## Admonitions
+
+Admonitions, also known as call-outs, are supported. Refer to the [official documentation](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#admonitions) for the full list of features and syntax. Note: You do not need to add the list of extensions to your mkdocs.yml file as described in the documentation. The extensions are included by default.
```

Comment:
I would move the comment re: not needing to add extensions and a link to https://squidfunk.github.io/mkdocs-material/reference/ to the introduction.
I'm not totally clear on the pymdown vs mkdocs distinction, but these docs seem to relate to what we currently have in place while https://facelessuser.github.io/pymdown-extensions/ does not.

Reply:
I've removed the comment as suggested. The top of the page includes links to the external documentation.